docs/docs/setup-vast/install/macos.md

```diff
@@ -0,0 +1,16 @@
+# macOS
```

Comment:
@dominiklohmann should we add Homebrew instructions here as well?

Reply:
That no longer exists. It's just building from source now. Installing via CMake also installs a launchd config file that can be enabled via `launchctl load -w ~/Library/LaunchAgents/com.tenzir.vast.plist`, but that's entirely optional.

Reply:
Can you provide me snippet/diff that I can apply to this branch?

Reply:
I'd just append one paragraph at the bottom:

> ## launchd
> 
> Installing VAST via CMake on macOS installs a [launchd agent](https://www.launchd.info) at `~/Library/LaunchAgents/com.tenzir.vast.plist`. To run VAST automatically on login, run this command:
> ```bash
> # NOTE: Use 'unload' rather than 'load' to unload the agent
> launchctl load -w ~/Library/LaunchAgents/com.tenzir.vast.plist
> ```